src/tests/ftest/soak/soak.py

```diff
@@ -308,33 +378,48 @@ def job_setup(self, test_param, pool):
                 "<<FAILED: There are only {} client nodes for this job. "
                 "Job requires {}".format(
                     len(self.hostlist_clients), self.nodesperjob))
-        if self.job_manager == "slurm":
+
+        # Prepend the  with cmdline to start daos agent
+
+        if "ior" in job:
+            # job_cmd_tasks = [[job, command, #tasks per node],
+            job_cmds_tasks = self.create_ior_cmdline(job, pool)
+        elif "dmg" in job:
+            # create dmg cmdline
+            job_cmds_tasks = self.create_dmg_cmdline(job, pool)
```

Comment:
(pylint-redefined-variable-type) Redefinition of job_cmds_tasks type from list to str